renderer-macos/lib/components/Clickable.re

```diff
@@ -0,0 +1,35 @@
+open Brisk;
+
+type attribute = [ Layout.style | Styles.viewStyle];
+
+type style = list(attribute);
+
+let component = {
+  let component = nativeComponent("clickable");
+  (~style=[], ~onClick=() => (), ~children: list(Brisk.syntheticElement), ()) =>
+    component(hooks =>
+      (
+        hooks,
+        {
+          make: () => {
+            let view = BriskClickable.make(~onClick, ());
+            let layoutNode =
+              Layout.Node.make(~style, {view, isYAxisFlipped: false});
+
+            {view, layoutNode};
+          },
+          configureInstance: (~isFirstRender as _, {view} as node) => {
+            style
+            |> List.iter(attribute =>
```

Comment:
This boilerplate is everywhere. We'll have to think what to do about it. Could you add an issue?

Reply:
Filed under #77.